import java.io.IOException;

public class FabonacciSeries {
	
	public static void main( String[] str)
	{
		
		try
		{
			System.out.println("try block");
		}
		
		catch(Exception e)
		{
			
		}
		finally {
			System.out.println("This is finally block");
		}
		
	}
	

}
